languages/rust/exercises/concept/entry-api/.docs/instructions.md

```diff
@@ -0,0 +1,24 @@
+# Instructions
+
+In this exercise you'll be using a `HashMap`, along with entry API methods, to solve a simple algorithm problem.
+
+The problem states: given a slice of `&str`s representing a magazine and another slice of `&str`s representing a note you're looking to construct, implement a function that returns a `bool` indicating whether the note can be constructed from the magazine. In other words, does the magazine contain at least as many instances of each word required by the note?
+
+You'll start with the following stubbed function signature:
+
+```rust
+pub fn construct_note(magazine: &[&str], note: &[&str]) -> bool {
```

Comment:
```suggestion
pub fn can_construct_note(magazine: &[&str], note: &[&str]) -> bool {
```

Naming boolean methods `can_*` or `is_*` improves legibility, and prevents future conflicts with a future method which actually does the work of constructing the desired note.